### PR TITLE
move input query construction from preprocessor to inputquery

### PR DESCRIPTION
**Pull request by anwu1219**
This allows us to call constructNLR() after loading the .nnet file, so that we can load bounds of internal variables in the property file.

### CODE REVIEW

File: src/configuration/GlobalConfiguration.cpp [hunk 1]
```diff
@@ -89,7 +89,7 @@ const bool GlobalConfiguration::TABLEAU_LOGGING = false;
 const bool GlobalConfiguration::SMT_CORE_LOGGING = false;
 const bool GlobalConfiguration::DANTZIGS_RULE_LOGGING = false;
 const bool GlobalConfiguration::BASIS_FACTORIZATION_LOGGING = false;
-const bool GlobalConfiguration::PREPROCESSOR_LOGGING = false;
+const bool GlobalConfiguration::INPUT_QUERY_LOGGING = true;
```
Comment (guykatzz) [inline src/configuration/GlobalConfiguration.cpp:92]: Set to false by default?

Comment (anwu1219) [reply src/configuration/GlobalConfiguration.cpp]: Done!

**Comment by guykatzz** [comment]
LGTM

File: src/input_parsers/PropertyParser.cpp [hunk 2]
```diff
@@ -138,7 +138,7 @@ void PropertyParser::processSingleLine( const String &line, InputQuery &inputQue
 
             auto subToken = subTokens.begin();
             ++subToken;
-            unsigned layerIndex = atoi( subToken->ascii() );
+            unsigned layerIndex = 2 * atoi( subToken->ascii() ) - 1;
```
Comment (guykatzz) [inline src/input_parsers/PropertyParser.cpp:141]: Hmm...
So, it seems like this change is tightly coupled to the ACAS Xu architecture, of input, <fc,relu>*, output layers. Our NLR now supports general feed-forward topologies. I understand the motivation was to maintain the old ws indexing, but I'm not sure it's good to hard-code this.

**Pull request merged**